pkg/apis/v1alpha/monitoring/monitor_handler.go

```diff
@@ -0,0 +1,361 @@
+/*
+Copyright 2018 The KubeSphere Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package monitoring
+
+import (
+	"encoding/json"
+	"regexp"
+	"strings"
+
+	"github.com/bitly/go-simplejson"
+	"github.com/emicklei/go-restful"
+	"github.com/emicklei/go-restful-openapi"
+	"github.com/golang/glog"
+
+	"kubesphere.io/kubesphere/pkg/client"
+	"kubesphere.io/kubesphere/pkg/filter/route"
+	"kubesphere.io/kubesphere/pkg/models/metrics"
+)
+
+func monitorWorkLoadSingleMetric(request *restful.Request, metricsName string) *simplejson.Json {
```

Comment:
Normally, you should write register and handler in this package and write processing code in `models` package.